json-endpoints/jetpack/class.jetpack-json-api-themes-endpoint.php

```diff
@@ -100,7 +101,8 @@ protected function format_theme( $theme ) {
 		$id = $theme->get_stylesheet();
 		$formatted_theme = array(
 			'id'          => $id,
-			'screenshot'  => jetpack_photon_url( $theme->get_screenshot(), array(), 'network_path' )
+			'screenshot'  => jetpack_photon_url( $theme->get_screenshot(), array(), 'network_path' ),
+			'active'      => $id === $this->current_theme_id,
```

Comment:
I was wondering about using an inline expression in 5.2, but I checked and we're all :+1: